keyboards/augio/readme.md

```diff
@@ -0,0 +1,32 @@
+i<pre>
```

Comment:
This file should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).